### ISSUE TITLE
'Clip Start' not quite accurate

### ISSUE THREAD

**Issue by Anonymousy** [ozmartian/vidcutter#31]
If I select 'Clip Start' at the beginning of a video, the start point works fine.  If I click 'Clip Start' later in the video it is very close, but not immediate and after a couple seconds go by then it marks the start point.   

'Clip End' button always works well, no problem with selecting the end point of the video.

Also, the program doesn't remember the last opened file location. Which isn't a bug, more of a user preference.

**Comment by SelfRando** [comment]
I agree - this is a major problem for me because I am trying to cut short clips. 
I end up with a pause without any video for about 3 seconds followed by maybe one or two seconds of action before the end of the clip.

My research has discovered that this is expected behaviour for certain types of input file when using the 'copy' feature of fmpeg (i.e. the input file is not decode coded and reencoded)
http://stackoverflow.com/a/18449609
https://trac.ffmpeg.org/wiki/Seeking

To work around this issue I have been using vidcutter to identify the start and end points I want for my clip and then running ffmpeg on the command line  :-(

Here are my commands:
ffmpeg -i source.avi -ss 00:30:49.640 -to 00:30:55.712 -q:v 0 -async 1 cut.mp4

This next command makes the cutting process faster by jumping over the first 30 minutes and then seeking to the start of the cut. Note that you have to adjust the --ss and -to times to take account of the missing 30 minutes:

ffmpeg -ss 00:30:00.000 -i source.avi -ss 00:00:49.640 -to 00:00:55.712 -q:v 0 -async 1 cut.mp4

Hope that helps!

**Comment by felagund** [comment]
Yes, I am running into this too.

In principle, shouldn't it be possible ot reencode just the bit until the first keyframe in the selection and then merge it with the original video as is - all the setting would have to be taken from the video first.

**Comment by SelfRando** [comment]
FYI, this has not changed in version 3.5.0

**Comment by felagund** [comment]
Yes, in settings, one can enable SmartCut and it works as expected. This issue can be closed.
